### PR TITLE
Budget-App

### PR DESCRIPTION
**Pull request by mwanawabangona**
In this project update:

Interactions
 Create the following interactions:
Splash screen
Sign up and log in pages
Home page (catagories page)
Transactions page
"Add a new category" page
"Add a new transaction" page
Testing requirements
 Create unit and integration tests for all the most important components of the RoR application.
Technical requirements
 Use Postgres as database.
 Use devise for authentication.
 Validate all user input to make sure that anyone with bad intentions cannot - compromise the app.
 Use a view template engine of your choice (.erb, .slim, .haml).
 The project should be deployed and accessible online.
 Database schema reflects the ERD diagram

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -42,180 +43,198 @@
...
 .background span:nth-child(18) {
-  color: #E45A84;
+  color: #e45a84;
   top: 94%;
   left: 88%;
   animation-duration: 57s;
   animation-delay: -100s;
   transform-origin: -22vw 7vh;
   box-shadow: -58vmin 0 7.9246528170907vmin currentColor;
 }
```
Comment (Emmyn5600) [inline app/assets/stylesheets/application.css:234]: [ ] 3 tests are failing, kindly work on them to pass successfully.

- image

[ ] Kindly, make sure that text `< class="d-grid gap-2 col-6 mx-auto">` won't appear before save button. 

- image

**Review by Emmyn5600 — COMMENTED** [review]
Hi @mwanawabangona 

Good job so far! I know you have worked hard on making this milestone. There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
          - Highlights
- Database is well structured.
- The readme file is descriptive
- You have descriptive PR. 
- The UI looks great
- You have completed almost all the requirements.

## Required Changes :recycle:
- [ ] Check the comments under the review too.
- [ ] Live version


- Deploy the application using Heroku, Netlify, or GitHub Pages.
- The project is deployed properly, and the home and detail page works as expected.
- A link to the deployed project is present in the README.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!:clap::clap::clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (mwanawabangona) [reply app/assets/stylesheets/application.css:234]: @Emmyn5600 Thank you for your review. I have pushed the required changes

**Review by Cathella — COMMENTED** [review]
## 🔥 Status: Approved 🟢

### Hi @mwanawabangona 👋🏻, 

This app looks great, I love everything about it 😍. You did a very nice job 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**